components/esp_wifi/Kconfig

```diff
@@ -562,4 +562,56 @@ menu "Wi-Fi"
         help
             Select this to enable unity test for DPP.
 
+
+    menu "WiFi SoftAP specific Network address"
```

Comment:
hi @MarcFinetRtone 

The proposed change is put under this path.

components/lwip/Kconfig